react-bootstrap/react-bootstrap.d.ts

```diff
@@ -704,6 +704,88 @@ declare module "react-bootstrap" {
     }
     var FormControls: FormControlsClass;
 
+    // <Form />
+    // ----------------------------------------
+    interface FormProps extends React.HTMLAttributes {
+        bsClass?: string;
+        componentClass?: string;
+        horizontal?: boolean;
+        inline?: boolean;
+    }
+    type Form = React.ClassicComponent<FormProps, {}>;
+    var Form: React.ClassicComponentClass<FormProps>;
```

Comment:
[`Form`](https://github.com/react-bootstrap/react-bootstrap/blob/master/src/Form.js) is a modern component, not classic